packaging/fedora/build

```diff
@@ -162,16 +185,22 @@ def main():
       cmd_ok(container, "rpm", "--erase", PACKAGES)
       print("# installing")
       for p in PACKAGES:
-         cmd(container, "rpm", "--install",
-             "%s/%s-%s.*.rpm" % (rpms, p, rpm_vr))
+         if p == 'charliecloud-doc':
+            file_ = "%s/%s-%s.%s.noarch.rpm" % (rpms_noarch, p, rpm_vr, rpm_dist)
+         else:
+            file_ = "%s/%s-%s.%s.%s.rpm" % (rpms, p, rpm_vr, rpm_dist, ARCH)
+         cmd(container, "rpm", "--install", file_)
       cmd(container, "rpm", "-qa", "charliecloud*")
```

Comment:
Yes. This is because I need to bind `git_tmp` at `/mnt/0` and instantiate the process there. Since `git_tmp` is removed after the configure step, including `-b %s --cd /mnt/0` in the container list would cause a failure here.

I didn't see a clean way to merge both use cases into one variable, so I left `container`  alone and just used CH_RUN above.